src/styles/annotator/inject.scss

```diff
@@ -9,6 +9,13 @@ $base-font-size: 14px;
 .annotator-highlights-always-on {
   .annotator-hl {
     background-color: $highlight-color;
+
+    // cursor pointer gives mobile devices click
+    // event support. Using the pointer:coarse gives
+    // us good targeting to mobile devices that this effects
```

Comment:
I know what you mean by "good targeting" but the wording seems a bit odd. I would have said "limits this to mobile devices".